pkg/services/anonymous/anonimpl/anonstore/database.go

```diff
@@ -13,10 +13,14 @@ import (
 )
 
 const cacheKeyPrefix = "anon-device"
+const thirtyDays = 30 * 24 * time.Hour
```

Comment:
Question: is this the timeframe that we have agreed on for counting anonymous devices as active? I feel like it might be worth to add a comment here or maybe rename the variable to something like `anonymousDeviceLifetime` (or something more accurate) to make it easier to understand why 30 days are used.

Reply:
30 days was chosen to be in line with the '[Active users](https://grafana.com/docs/grafana/latest/administration/enterprise-licensing/#active-users-limit)' definition.

Reply:
@IevaVasiljeva your suggestion to make this variable more clear `anonymousDeviceLifetime` is noted. We would want this to be in the other places as well. We have this in the `api.go` as well.

Add a note for my self
```
// FIXME: rename thirtyDays to something more descriptive, like "anonymousDeviceLifetime", or "anonymousDeviceExpiration"
```